socorro/external/rabbitmq/crashstorage.py

```diff
@@ -79,40 +63,18 @@ def __init__(self, config, namespace='', quit_check_callback=None):
             quit_check_callback=quit_check_callback
         )
 
-        self.config = config
```

Comment:
This gets set in the superclass `__init__`, so we don't need to do it again here.